companies/bancopichincha-es.json

```diff
@@ -0,0 +1,37 @@
+{
+    "slug": "bancopichincha-es",
+    "relevant-countries": [
+        "es"
+    ],
+    "categories": [
+        "finance"
+    ],
+    "name": "Banco Pichincha España S.A",
+    "address": "Calle Lagasca 4\n28001, Madrid\nEspaña",
+    "phone": "+34 914 36 25 94",
+    "email": "dpo@bancopichincha.es",
```

Comment:
As this email is specific for privacy-related requests, we can set the `suggested-transport-medium` to `email`.